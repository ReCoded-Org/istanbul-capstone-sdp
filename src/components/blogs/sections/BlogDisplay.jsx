import React from "react";
import { useTranslation } from "react-i18next";
import { Col, Row } from "react-bootstrap";
import "../index.css";
export default function BlogDisplay(props) {
  const { t } = useTranslation();
  return (
    <>
      <Col className="oneLineBlogpost ">
        <Row>
          <Col xs={12} md={6} className="blogPhotoContainer">
            <img
              className="blogPic"
              src={props.photo}
              alt="Blog Post Content"
            />
            <br />
          </Col>
          <Col xs={12} md={6} className="blogPhotoTextContent">
            <h4>{t("blogposts." + props.translationKey + ".title")}</h4>
            <p>{t("blogposts." + props.translationKey + ".desc")}</p>
          </Col>
        </Row>
      </Col>
    </>
  );
}
