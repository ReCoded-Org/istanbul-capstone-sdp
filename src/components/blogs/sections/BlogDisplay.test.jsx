import React from "react";
import BlogDisplay from "./BlogDisplay";
import { render } from "@testing-library/react";

it("blog display snapshot", () => {
  const tree = render(<BlogDisplay />);
  expect(tree).toMatchSnapshot();
});
