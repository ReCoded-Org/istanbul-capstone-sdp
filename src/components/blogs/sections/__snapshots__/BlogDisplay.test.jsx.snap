// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blog display snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="oneLineBlogpost  col"
      >
        <div
          class="row"
        >
          <div
            class="blogPhotoContainer col-md-6 col-12"
          >
            <img
              alt="Blog Post Content"
              class="blogPic"
            />
            <br />
          </div>
          <div
            class="blogPhotoTextContent col-md-6 col-12"
          >
            <h4>
              blogPosts.undefined.title
            </h4>
            <p>
              blogPosts.undefined.excerpt
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="oneLineBlogpost  col"
    >
      <div
        class="row"
      >
        <div
          class="blogPhotoContainer col-md-6 col-12"
        >
          <img
            alt="Blog Post Content"
            class="blogPic"
          />
          <br />
        </div>
        <div
          class="blogPhotoTextContent col-md-6 col-12"
        >
          <h4>
            blogPosts.undefined.title
          </h4>
          <p>
            blogPosts.undefined.excerpt
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
